src/hammer-vlsi/defaults.yml

```diff
@@ -819,6 +827,30 @@ sim.inputs:
   # If set to True, the sim will be time annotated based on synthesis or PAR results.
   timing_annotated: false
 
+  # Settings related to activity file (SAIF) dumping
+  saif:
+    # Enable dumping an SAIF file, with activity factors
+    # type: bool
+    enable: false
```

Comment:
Definitely more of a nit, but based on the logic in the companion PR, it appears to me that you could combine this `enable` flag with the `mode` key a certain way to maybe be less confusing to the user. Perhaps `none` below should be `full` and `none` or anything else that's not a valid option will not dump the SAIF file.

Reply:
Yeah that seems reasonable and matches more of what the other APIs look like.
I think this weirdness stems from the evolution of the API as this PR grew in size.
Will fix soon.

Reply:
Fixed.